build.zig

```diff
@@ -48,6 +50,22 @@ pub fn build(b: *zbs.Builder) !void {
 
     const examples = b.option(bool, "examples", "Set to true to build examples") orelse false;
 
+    const full_version = blk: {
+        if (mem.endsWith(u8, version, "-dev")) {
+            var ret: u8 = undefined;
+            const git_dir = fs.path.join(b.allocator, &[_][]const u8{ b.build_root, ".git" }) catch unreachable;
+            const git_commit_hash = b.execAllowFail(
+                &[_][]const u8{ "git", "--git-dir", git_dir, "--work-tree", b.build_root, "rev-parse", "--short", "HEAD" },
+                &ret,
+                .Ignore,
```

Comment:
I think this is more correct so that anything `git` logs to stderr will still be printed.
```suggestion
                .Inherit,
```